docs/content/glossary/glossary.mdx

```diff
@@ -0,0 +1,318 @@
+---
+title: Glossary
+---
+
+# Flow Glossary
+
+## All words and definitions related to Blockchain and Flow
+
+## **A**
+
+## Authentication
+[Authentication](https://docs.onflow.org/fcl/reference/authentication/#gatsby-focus-wrapper) connects users to their wallets, 
+and the wallet provides the account information and services through FCL.
```

Comment:
I'd say auth is the mechanism to verify that a given user is who they claim to be. Wallets perform this check. FCL facilitates auth on Flow

Reply:
fixed